packages/react-composites/src/composites/ChatComposite/file-sharing/FileCardGroup.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @beta
+ */
+export interface FileCardGroupProps {
+  children: React.ReactNode;
+}
+
+/**
+ * @beta
+ */
+export const FileCardGroup = (props: FileCardGroupProps): JSX.Element => {
+  const { children } = props;
+  return (
+    <Stack
+      horizontal
+      className={mergeStyles({
+        flexFlow: 'row wrap',
```

Comment:
instead of defining naked css, can this be something like `<Stack horizonal wrap tokens={ { childrenGap: '0.5rem 0.5rem' } }  />` (not sure exactly what it should be)

Reply:
Right. I was using childrenGap originally, but it doesn't work as expected for our designed. When child elements are wrapped and moved to the next line, the styling breaks unless done like this ;)

Reply:
Please add this comment to the code for the next reader.

Reply:
Will do